### PR TITLE
Update Leland to version 0.76

### PR DESCRIPTION
**Pull request by cbjeukendrup**

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

File: fonts/leland/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Leland music fonts
 
-This repository contains the **Leland music fonts, Version 0.74**.
+This repository contains the **Leland music fonts, Version 0.75**.
 
 ## About the fonts
 
-The Leland music fonts (Leland & Leland Text), Version 0.74 are available in Type 1, OpenType-CFF format. They are distributed under the [SIL Open Font License (OFL), Version 1.1](./LICENSE.txt).
+The Leland music fonts (Leland & Leland Text), Version 0.75 are available in Type 1, OpenType-CFF format. They are distributed under the [SIL Open Font License (OFL), Version 1.1](./LICENSE.txt).
```
Comment (Jojo-Schmitz) [inline fonts/leland/README.md:7]: and here too

File: fonts/leland/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Leland music fonts
 
-This repository contains the **Leland music fonts, Version 0.74**.
+This repository contains the **Leland music fonts, Version 0.75**.
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply fonts/leland/README.md:3]: This here is something @oktophonie forgot too

[the remark this answers is not in the document]
Comment (oktophonie) [reply fonts/leland/README.md:3]: > I guess the vtest diffs are to be expected?

Do you?

**Comment by oktophonie** [comment]
> I guess the vtest diffs are to be expected?

Do you?

File: fonts/leland/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oktophonie) [reply fonts/leland/README.md:3]: Crisis. MU4 release cancelled.

**Pull request merged**